### PR TITLE
New features to navbar

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #80 #89 

This is how the navbar looks on pages where the background is entirely white
screenshot from 2017-07-18 15 11 39

And this is how it looks on the home page
screenshot from 2017-07-18 15 16 13

@lucyrose93 is the color how you wanted it ?

### CODE REVIEW

File: src/views/search_results.hbs [hunk 1]
```diff
@@ -1,4 +1,3 @@
-{{>navbar}}
```
Comment (Karyum) [inline src/views/search_results.hbs]: why did you remove it from here?

File: src/views/startup_profile.hbs [hunk 2]
```diff
@@ -1,4 +1,3 @@
-{{>navbar}}
```
Comment (Karyum) [inline src/views/startup_profile.hbs]: and this?

File: src/views/layouts/main.hbs [hunk 3]
```diff
@@ -1,3 +1,4 @@
+{{>navbar}}
```
Comment (Karyum) [inline src/views/layouts/main.hbs]: lets keep the file naming consistent with kabab case ( nav-bar instead of navbar )

File: src/views/search_results.hbs (hunk 1 above)
Comment (Suuuuha) [inline src/views/search_results.hbs]: Because we need it on all the pages now so I added it on the home layout !

File: src/views/layouts/main.hbs (hunk 3 above)
Comment (Suuuuha) [inline src/views/layouts/main.hbs]: navbar is a one word for me :nerd_face:

File: src/views/search_results.hbs (hunk 1 above)
Comment (Karyum) [inline src/views/search_results.hbs]: alright

File: src/views/layouts/main.hbs (hunk 3 above)
Comment (Karyum) [inline src/views/layouts/main.hbs]: navbar is 2 words nav and bar so why not kabab case?

**Comment by Suuuuha** [comment]
@Karyum I've made changes

**Comment by lucyrose93** [comment]
@Suuuuha @Karyum 
That is what I meant, yes. The home page nav bar can remain white, as it looks better with the indigo banner. 
Also, in the initial designs, we had "startups" in orange, rather than "join us". 
The idea was to encourage site visitors to look at our startups.  
@esraajb do you think we should draw attention to "join us" or "startups" in the nav bar?

File: public/form.js [hunk 4]
```diff
@@ -3,3 +3,6 @@ var coFounderInput = document.querySelector('#coFounder-name');
 plusCircle.addEventListener('click', function () {
   coFounderInput.classList.toggle('dn');
 })
+
+var bgChangeColor = document.querySelector('#navbar');
```
Comment (esraajb) [inline public/form.js]: use let

File: public/form.js [hunk 5]
```diff
@@ -3,3 +3,6 @@ var coFounderInput = document.querySelector('#coFounder-name');
 plusCircle.addEventListener('click', function () {
```
Comment (esraajb) [inline public/form.js]: use es6 function

File: public/style.css [hunk 6]
```diff
@@ -29,6 +29,10 @@
 	color: #5d5b6d;
 }
 
+.bg-navbar{
+  color: #9B9B9B;
```
Comment (esraajb) [inline public/style.css]: the name of the class is background navbar, why not move the color to a different class ?

File: src/views/partials/navbar.hbs [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<nav class="dt w-100 bg-navbar center" id="navbar">
+  <div class="dtc w2 v-mid ">
+      <a class="f1  b no-underline dark-gray mh3" href="/">Ain</a>
+  </div>
+  <div class="dtc v-mid tr">
```
Comment (esraajb) [inline src/views/partials/navbar.hbs]: do we need the v-mid?

File: public/form.js (hunk 5 above)
Comment (Karyum) [inline public/form.js]: this was in an old PR that was approved
and this is the front end we cant use es6 its not supported in all browsers

File: public/form.js (hunk 4 above)
Comment (Karyum) [inline public/form.js]: again some browsers don't support it

File: src/views/partials/navbar.hbs (hunk 7 above)
Comment (Suuuuha) [inline src/views/partials/navbar.hbs]: In the first div no i just removed it, but in the second yes to keep the links(tabs) in the center of the div

File: public/style.css (hunk 6 above)
Comment (Suuuuha) [inline public/style.css]: I thought it will be more clear since we are using this color only in the navbar background with 10%
